stdlib/typing_extensions.pyi

```diff
@@ -200,6 +205,8 @@ else:
     def reveal_type(__obj: _T) -> _T: ...
     def assert_never(__arg: NoReturn) -> NoReturn: ...
     def assert_type(__val: _T, __typ: Any) -> _T: ...
+    def clear_overloads() -> None: ...
+    def get_overloads(func: Callable[..., object]) -> Sequence[Callable[..., object]]: ...
 
 # Experimental (hopefully these will be in 3.11)
```

Comment:
I guess this comment is now outdated?

Reply:
For most of them, dataclass_transform is still pending. I'll update this.